js/newSubmit.js

```diff
@@ -1322,24 +1326,22 @@ Do you still want to save this draft?`)) return;
                             }
                             dialogData.loading.css("display", "none");
                             currentVersions = versions;
-                            var first = true;
                             dialogData.table.find("tr.submit-deps-version-select-row").remove();
                             for(var relId in versions) {
                                 if(!versions.hasOwnProperty(relId)) continue;
                                 var dateSpan = $("<span></span>").attr("data-timestamp", versions[relId].submitTime);
                                 timeTextFunc.call(dateSpan.get()[0]);
                                 // noinspection JSUnresolvedVariable
-                                dialogData.table.append($("<tr></tr>").addClass("submit-deps-version-select-row")
+                                dialogData.table.prepend($("<tr></tr>").addClass("submit-deps-version-select-row")
                                     .append($("<td></td>")
                                         .append($("<input type='radio'/>").addClass("submit-dep-dialog-version-radio").attr("name", versionDialogLock)
                                             .attr("data-value", relId)
-                                            .prop("checked", first))
+                                            .prop("checked", true))
```

Comment:
Won't this check every radio button?

Reply:
Not if they are grouped as only one per radio group can be checked - this leaves only the latest selected